index.js

```diff
@@ -1,18 +1,44 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = 'yellow'
+header.style.background = 'blue'
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+// name
+let travelerName = document.querySelector("h2")
+travelerName.textContent = `${traveler.name}`
+
+// nickname
+let travelerNickname = document.querySelector("em")
+travelerNickname.textContent = `${traveler.nickname}`
+
+// image WITH src and alt
+
+// let travelerImageSrc = document.querySelector("img").src 
+// travelerImageSrc.textContent = `${traveler.photo}`
+
+document.querySelector("img").src = `${traveler.photo}`
+
+// The commented out version is how I feel like this should work, but it isn't working. I can only 
+// get this to work with the above line of code, which feels icky to me. Is this what it should be?
+
+document.querySelector("img").alt = `${traveler.nickname}`
```

Comment:
The reason this line doesn't work: 

```js
let travelerImageSrc = document.querySelector("img").src 
```

is because you're finding the element, and then accessing its 'src' attribute, and saving its `src` to a variable. You want to save the whole DOM element to a variable, like this:

```js
let travelerImage = document.querySelector("img")
```

In which case you can then set its `src` and `alt` attributes:

```js
travelerImage.src = traveler.photo
travelerImage.alt = traveler.nickname
```